app/scripts/lib/app-start.js

```diff
@@ -270,6 +274,12 @@ function (
       });
     },
 
+    initializeMarketingEmailClient: function () {
```

Comment:
@zaach - Except for fxa-client, I don't think any of these clients save any state. Instead of creating all these clients on startup and passing them around, we could just pass in a list of service urls and have the callers of the clients create them when needed? Dunno.